controllers/nnf_workflow_controller_helpers.go

```diff
@@ -250,19 +250,35 @@ func (r *NnfWorkflowReconciler) validateContainerDirective(ctx context.Context,
 		return fmt.Errorf("storage '%s' not found in container profile '%s'", storageName, profile.Name)
 	}
 
+	checkContainerFs := func(idx int) error {
+		getDirectiveFsType := func(idx int) string {
+			args, _ := dwdparse.BuildArgsMap(workflow.Spec.DWDirectives[idx])
+			return args["type"]
+		}
+
+		if t := getDirectiveFsType(idx); strings.ToLower(t) == "raw" || strings.ToLower(t) == "xfs" {
```

Comment:
Maybe check the inverse of this instead? `if type == "lustre" || type == "gfs2"`. That makes it less likely to cause a bug if we add a new file system type in the future. It's up to you, though. We probably won't add anymore FS types.

Reply:
I actually had it this way to start and then waffled and switched to this. I'll change it back to look for lustre/gfs2.